### DISCUSSION TITLE
HDR brightness

### DISCUSSION THREAD

**Discussion by ncortines** [MonitorControl/MonitorControl#673] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Hi! this is a question.

Can MonitorControl help with brightness while using a display on HDR mode?

I recently purchased an external monitor with HDR support. While enabling HDR on Display settings, everything becomes much darker. Even shows with HDR support, are too dark.

Other observations:
- While HDR is enabled on the monitor, the OSD for brightness control is disabled.
- While trying to adjust brightness using MonitorControl while HDR is enabled, the display becomes brighter for the split of a second, then goes back to dark.

Thanks!
Juan

**Comment by waydabber** [comment, marked as answer]
Well, if you purchased the common VESA DisplayHDR 400 certified LCD display, then it makes no sense to use HDR at all. The only thing this achieves is that the desktop will be dimmed singificantly as the OS will reserve much of the dynamic range for HDR luminance that should be over the SDR luminance range + in case of some HDR videos in players that support HDR, the video box plaing with HDR content will not be dimmed. You are better off disabling HDR and using players (like IINA) that can map HDR content to SDR. The result will be basically the same in terms of the video content but the display will not be dimmed.

Most VESA 400 so called 'HDR displays' do not allow any kind of brightness control in HDR mode, this is a hardware limitation, as in this mode backlight tends to work at 100%. You can use software dimming to further decrease the dynamic range though, but it makes no sense.

If you have a VESA 600 display, using HDR still makes no sense for normal desktop use, but these units usually have at least some controllable dimming zones (usually 8-16), so for full screen games and movies it at least makes some difference.

If you purchased a proper HDR display with mini-led or FALD backlighting (these usually marketed with a DisplayHDR 1000 badge) or you use an OLED TV display then it makes more sense to enable HDR generally (if HDR is implemented correctly and does not mess up desktop image clarity as some displays tend to do funny image processing when HDR is enabled). On these displays using software dimming also makes sense as when dimming occures, the display is clever enough to reduce the direct backlighting (or obviously pixel lighting for OLED) while increasing the LCD pixel intensity.

**Comment by waydabber** [comment]
I'll convert this one into to a Q&A entry.
